core/src/main/java/com/adobe/aem/commons/assetshare/content/properties/AbstractComputedProperty.java

```diff
@@ -59,23 +61,40 @@ protected ValueMap getMetadataProperties(Asset asset) {
         return metadataResource.getValueMap();
     }
 
+    protected ValueMap getJcrProperties(final Asset asset) {
```

Comment:
can `getAssetProperties().get("jcr:content/someProp")` suffice?
Else maybe better name this method to `getJcrContentProperties()`? `getJcrProperties()` could mean anything - theyre all JCR properties! :)